### DISCUSSION TITLE
Is this only open to students?

### DISCUSSION THREAD

**Discussion by josharian** [ethz-spylab/satml-llms-ctf-issues#5] [Questions]
The registration doesn't have an option for 0 students on a team.

**Comment by dedeswim** [comment]
Yes it is open to non-students. You can just not select any value from the form.

I hope this clarifies your doubts!

**Reply by josharian** [reply]
Thanks!

I already submitted with “1” selected. Maybe you can fix that. :) I’d suggest adding a zero option to avoid future confusion.

**Reply by dedeswim** [reply]
Done, thanks a lot for the suggestion!
